src/Registry/Registry.php

```diff
@@ -16,19 +16,19 @@ class Registry
     /**
      * List of all fresh resources
      *
-     * @var array
+     * @var array<bool>
```

Comment:
This also looks like it shouldn't be `array<bool>` considering it's used like `$this->fresh[$name] = $fresh;`

Reply:
resolved 👍